frontend/app/utils/route-utils.ts

```diff
@@ -1,3 +1,5 @@
+import { getLanguage } from './i18n-utils';
```

Comment:
To be consistent with the other imports, this should be `from '~/utils/i18n-utils'`